src/game_objects/game_over_dialog.py

```diff
@@ -4,28 +4,31 @@
 
 
 class GameOverDialog(GameObject):
-
-    def __init__(self, uptime, score, restart_game_fn, main_menu_fn):
+    # pylint: disable=too-many-arguments
```

Comment:
to support the extra `standalone` argument, the `too-many-arguments` error would appear (6/5)
(same as scene/game.py)